src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx

```diff
@@ -59,15 +59,15 @@ const AnonymousChannelListScreen = ({route}) => {
     <>
       <StatusBar translucent={false} />
       <View style={{height: 52}}>
-        <Search route={route} onPress={goToContactScreen} isShowNewChat={false} />
+        <Search route={route} isAnon={true} onPress={() => goToContactScreen({from: ANONYMOUS})} />
       </View>
 
       <FlatList
         data={anonChannels}
         keyExtractor={(item) => item.id}
         ListHeaderComponent={
           <ChannelListHeaderItem
-            name="Anonymous"
+            name="Incognito"
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The Search component props and ChannelListHeaderItem name were updated. The changes seem appropriate given the context of anonymous chats. However, ensure that the `goToContactScreen` function can handle the new payload `{from: ANONYMOUS}` correctly.
```diff
- <Search route={route} onPress={goToContactScreen} isShowNewChat={false} />
+ <Search route={route} isAnon={true} onPress={() => goToContactScreen({from: ANONYMOUS})} />

- <ChannelListHeaderItem name="Anonymous" picture={AnonymousProfile} type="ANONYMOUS" testID="horizontal-tab-1" />
+ <ChannelListHeaderItem name="Incognito" picture={AnonymousProfile} type="ANONYMOUS" testID="horizontal-tab-1" />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->